docs/tasks/ingress.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Exposing a Service with Ingress
+headline: Exposing a Service with Ingress
+sidenav: doc-side-tasks-nav.html
+bodyclass: docs
+layout: docs
+type: markdown
+---
+{% capture overview %}
+... overview of ingress as a concept ...
+
+This task describes how to configure Istio to expose a service in a Kubernetes cluster. You'll learn how to create an Ingress controller, define a Ingress Resource and make requests to the service.
+
+In a Kubernetes environment, Istio uses [Kubernetes Ingress Resources](https://kubernetes.io/docs/concepts/services-networking/ingress/) to configure ingress behavior.   
+
+{% endcapture %}
+
+{% capture prerequisites %}
+* `kubectl` and access to a Kubernetes cluster with Istio deployed in it. See (xxx)[].
+{% endcapture %}
+
+{% capture steps %}
+## Doing ...
+
+### Setup the environment
+Create an Istio Ingress controller.
+```yaml
+apiVersion: extensions/v1beta1
```

Comment:
Yes, istio-ingress is installed by default. Isaiah, you can add to prerequisites something like:

"Follow the Istio [Installation]({{site.bareurl}}/docs/tasks/istio-installation.html) guide".

BTW, I will rename istio-ingress-controller to istio-ingress.